src/components/GuideTile/GuideTile.js

```diff
@@ -16,33 +15,133 @@ const GuideTile = ({
   className,
   children,
   alignment,
+  featured,
   ...props
 }) => (
-  <Component
+  <Surface
     {...props}
-    className={cx(styles.tile, className, {
-      [styles.tileWithIcon]: icon,
-      [styles.tileLeftAligned]: alignment === GuideTile.ALIGNMENT.LEFT,
-      [styles.tileCenterAligned]: alignment === GuideTile.ALIGNMENT.CENTER,
-    })}
+    as={Component}
+    className={className}
+    base={Surface.BASE.PRIMARY}
+    css={css`
+      display: grid;
+      grid-template-rows: auto auto 1fr auto;
+      border-radius: 0.25rem;
+      position: relative;
+      padding: 1rem;
+      transition: all 0.15s ease-out;
+
+      &:hover {
+        transform: translateY(-2px);
+        border-color: var(--border-hover-color);
+      }
+
+      ${icon &&
+      `
+        h2 {
+          margin-top: 1rem;
+        }
+      `}
+
+      ${featured &&
+      `
+          border-color: var(--color-white);
+
+          .dark-mode & {
+            border-color: var(--color-dark-100);
+            background-color: var(--color-dark-100);
+          }
+
+          &:hover {
```

Comment:
Rather than unsetting the hover style for featured tiles, you can use the `interactive` prop to conditionally apply them. `interactive={!featured}`